Lib/typing.py

```diff
@@ -240,7 +243,8 @@ def _check_generic(cls, parameters, elen):
     alen = len(parameters)
     if alen != elen:
         raise TypeError(f"Too {'many' if alen > elen else 'few'} arguments for {cls};"
-                        f" actual {alen}, expected {elen}")
+                        f" actual {alen}, expected {elen} " + str(parameters))
```

Comment:
Was this a debugging change? It looks like it wouldn't create a nicely formatted error.

Reply:
Oops, good catch! Fixed.